src/ConfirmedMap.js

```diff
@@ -120,7 +120,7 @@ class MbMap extends React.Component {
                 flexDirection: 'column',
                 height: '520px'
             }}>
-                <h2>Hospitals & Cases Map</h2>
+                <h2>Hospitals & Case Map</h2>
```

Comment:
I think we may change it to Hospital & Case Map. you willing to do that change?

Reply:
Alright will do the change once more